doc/sphinx/addendum/extraction.rst

```diff
@@ -571,6 +571,12 @@ Additional settings
    This :term:`option` provides a comment that is
    included at the beginning of the output files.
 
+.. opt:: Extraction File Pragma @string
+
+   This :term:`option` provides a compiler pragma that is
+   included at the beginning of the output files.
+   ``Set Extraction File Pragma "OPTIONS_GHC -cpp -XMagicHash".``
```

Comment:
```suggestion
   Specifies a compiler pragma that is
   included at the beginning of the output files.
   For example, ``Set Extraction File Pragma "OPTIONS_GHC -cpp -XMagicHash".``
```

Based on your Haskell example, please explain that, IINM, the Pragma is embedded in a comment in the output.

We try to avoid using repetitive boilerplate such as "This option provides", though that still appears in many places.  The header in the generated doc will already label it as an option.

Not sure the example adds much.  Would be better if you very briefly said what `OPTIONS_GHS -cpp -XMagicHash` does.

Reply:
The example has been commented.